Unable to install OCaml variant 4.11.0+trunk

Sorry if this isn't the correct place to open the issue. `ocaml-beta-repository` didn't seem like the place since the `4.11.0+trunk` variant lives here in `ocaml-variants`?

These are the commands I've run:

```bash
opam switch create 4.11.0+trunk --empty
eval $(opam env)
opam repo add ocaml-beta https://github.com/ocaml/ocaml-beta-repository.git
opam install ocaml-variants=4.11.0+trunk
```

The last command produces this error:

```
The following actions will be performed:
  ∗ install base-bigarray  base
  ∗ install ocaml-beta     enabled      [required by ocaml-variants]
  ∗ install base-threads   base
  ∗ install base-unix      base
  ∗ install ocaml-variants 4.11.0+trunk
  ∗ install ocaml-config   1
  ∗ install ocaml          4.11.0
===== ∗ 7 =====
Do you want to continue? [Y/n] y

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
Processing  7/7: [ocaml-variants.4.11.0+trunk: http]
[ocaml-variants.4.11.0+trunk] downloaded from https://github.com/ocaml/ocaml/archive/4.11.tar.gz

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
∗ installed base-bigarray.base
∗ installed base-threads.base
∗ installed base-unix.base
∗ installed ocaml-beta.enabled
∗ installed ocaml-variants.4.11.0+trunk
∗ installed ocaml-config.1
[ERROR] The compilation of ocaml failed at "~/.opam/opam-init/hooks/sandbox.sh build ocaml ~/.opam/4.11.0+trunk/share/ocaml-config/gen_ocaml_config.ml 4.11.0
        ocaml".

#=== ERROR while compiling ocaml.4.11.0 =======================================#
# context     2.0.7 | linux/x86_64 |  | https://opam.ocaml.org#4f36a5bd
# path        ~/.opam/4.11.0+trunk/.opam-switch/build/ocaml.4.11.0
# command     ~/.opam/opam-init/hooks/sandbox.sh build ocaml ~/.opam/4.11.0+trunk/share/ocaml-config/gen_ocaml_config.ml 4.11.0 ocaml
# exit-code   1
# env-file    ~/.opam/log/ocaml-101563-d474c2.env
# output-file ~/.opam/log/ocaml-101563-d474c2.out
### output ###
# OCaml version mismatch: 4.11.1, expected 4.11.0



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
┌─ The following actions failed
│ λ build ocaml 4.11.0
└─ 
┌─ The following changes have been performed
│ ∗ install base-bigarray  base
│ ∗ install base-threads   base
│ ∗ install base-unix      base
│ ∗ install ocaml-beta     enabled
│ ∗ install ocaml-config   1
│ ∗ install ocaml-variants 4.11.0+trunk
└─ 

```

The OCaml 4.11 branch is currently on version 4.11.1, but I couldn't find a `4.11.1+trunk` variant.

--- a/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+afl/opam
+++ b/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+afl/opam
@@ -1,8 +1,8 @@
 opam-version: "2.0"
-synopsis: "latest 4.11.0 development, with afl-fuzz instrumentation"
+synopsis: "latest 4.11 development, with afl-fuzz instrumentation"
 maintainer: "platform@lists.ocaml.org"
 depends: [
-  "ocaml" {= "4.11.0" & post}
+  "ocaml" {= "4.11.1" & post}
   "base-unix" {post}
   "base-bigarray" {post}
   "base-threads" {post}

--- a/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+flambda/opam
+++ b/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+flambda/opam
@@ -1,8 +1,8 @@
 opam-version: "2.0"
-synopsis: "latest 4.11.0 development"
+synopsis: "latest 4.11 development, with flambda activated"
 maintainer: "platform@lists.ocaml.org"
 depends: [
-  "ocaml" {= "4.11.0" & post}
+  "ocaml" {= "4.11.1" & post}
   "base-unix" {post}
   "base-bigarray" {post}
   "base-threads" {post}
@@ -12,11 +12,12 @@ conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
-  ["./configure" "--prefix=%{prefix}%"]
+  ["./configure" "--prefix=%{prefix}%" "--enable-flambda"]
     {os != "openbsd" & os != "macos"}
   [
     "./configure"
     "--prefix=%{prefix}%"
+    "--enable-flambda"
     "CC=cc"
     "ASPP=cc -c"
   ] {os = "openbsd" | os = "macos"}

--- a/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+fp/opam
+++ b/packages/ocaml-variants/ocaml-variants.4.11.1+trunk+fp/opam
@@ -1,8 +1,8 @@
 opam-version: "2.0"
-synopsis: "latest 4.11.0 development, with flambda activated"
+synopsis: "latest 4.11 development, with afl-fuzz instrumentation"
 maintainer: "platform@lists.ocaml.org"
 depends: [
-  "ocaml" {= "4.11.0" & post}
+  "ocaml" {= "4.11.1" & post}
   "base-unix" {post}
   "base-bigarray" {post}
   "base-threads" {post}
@@ -12,14 +12,14 @@ conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
-  ["./configure" "--prefix=%{prefix}%" "--enable-flambda"]
+  ["./configure" "--prefix=%{prefix}%" "--with-afl" "--enable-frame-pointers" ]
     {os != "openbsd" & os != "macos"}
   [
     "./configure"
     "--prefix=%{prefix}%"
-    "--enable-flambda"
     "CC=cc"
     "ASPP=cc -c"
+    "--enable-frame-pointers"
   ] {os = "openbsd" | os = "macos"}
   [make "-j%{jobs}%" {os != "cygwin"} "world"]
   [make "-j%{jobs}%" {os != "cygwin"} "world.opt"]

--- a/packages/ocaml-variants/ocaml-variants.4.11.1+trunk/opam
+++ b/packages/ocaml-variants/ocaml-variants.4.11.1+trunk/opam
@@ -1,8 +1,8 @@
 opam-version: "2.0"
-synopsis: "latest 4.11.0 development, with afl-fuzz instrumentation"
+synopsis: "latest 4.11 development"
 maintainer: "platform@lists.ocaml.org"
 depends: [
-  "ocaml" {= "4.11.0" & post}
+  "ocaml" {= "4.11.1" & post}
   "base-unix" {post}
   "base-bigarray" {post}
   "base-threads" {post}
@@ -12,14 +12,13 @@ conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
-  ["./configure" "--prefix=%{prefix}%" "--with-afl" "--enable-frame-pointers" ]
+  ["./configure" "--prefix=%{prefix}%"]
     {os != "openbsd" & os != "macos"}
   [
     "./configure"
     "--prefix=%{prefix}%"
     "CC=cc"
     "ASPP=cc -c"
-    "--enable-frame-pointers"
   ] {os = "openbsd" | os = "macos"}
   [make "-j%{jobs}%" {os != "cygwin"} "world"]
   [make "-j%{jobs}%" {os != "cygwin"} "world.opt"]

